packages/langium-cli/src/generator/module-generator.ts

```diff
@@ -21,10 +21,16 @@ export function generateModule(grammar: langium.Grammar, config: LangiumConfig):
     node.append(
         'import { ', grammar.name, "AstReflection } from './ast';", NL,
         'import { ', grammar.name, "GrammarAccess } from './grammar-access';", NL,
-        'import { ', grammar.name, "LanguageMetaData } from './meta-data';", NL,
         "import { Parser } from './parser';", NL, NL
     );
 
+    node.append('const metaData = {', NL);
+    node.indent(metaData => {
+        metaData.append(`languageId: '${config.languageId}',`, NL);
+        metaData.append(`fileExtensions: [${config.extensions && config.extensions.map(e => appendQuotesAndDot(e)).join(', ')}]`, NL);
```

Comment:
Seeing this I think we could rename `extensions` to `fileExtensions` in the config as well, WDYT?

Reply:
Yeah, that makes sense.